# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libgpg-error/t2-linux.patch
# Copyright (C) 2018 - 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- libgpg-error-1.33/src/mkheader.c.vanilla	2019-01-15 19:00:36.878721125 +0000
+++ libgpg-error-1.33/src/mkheader.c	2019-01-15 19:00:47.046721009 +0000
@@ -91,6 +91,10 @@
     const char *name;
     const char *alias;
   } tbl[] = {
+    {"i486-unknown-linux-gnu", "i686-unknown-linux-gnu" },
+    {"i586-unknown-linux-gnu", "i686-unknown-linux-gnu" },
+    {"x86_64-unknown-linux-muslx32", "x86_64-unknown-linux-gnux32" },
+
     {"i486-pc-linux-gnu", "i686-unknown-linux-gnu" },
     {"i586-pc-linux-gnu" },
     {"i686-pc-linux-gnu" },
@@ -119,6 +119,12 @@
     {"armv5-unknown-linux-musleabi"   },
     {"armv6-unknown-linux-musleabihf" },
 
+    {"aarch64-unknown-linux-gnueabi", "aarch64-unknown-linux-gnu" },
+
+    {"mips64-unknown-linux-gnu", "mips64el-unknown-linux-gnuabi64" },
+    {"mips64-unknown-linux-gnuabi64", "mips64el-unknown-linux-gnuabi64" },
+    {"mips64-unknown-linux-gnuabin32", "mips-unknown-linux-gnu" },
+
     { NULL }
   };
   int i;
